### PR TITLE
Add --limit arg to similarity for fast testing, bump semsimian to 0.2.18

### PR DESCRIPTION
**Pull request by justaddcoffee**

### CODE REVIEW

**Comment by justaddcoffee** [comment]
@caufieldjh @hrshdhgd

File: src/oaklib/cli.py [hunk 1]
```diff
@@ -2511,75 +2513,34 @@ def similarity_pair(terms, predicates, autolabel: bool, output: TextIO, output_t
...
-
-    Python API:
-
-       https://incatools.github.io/ontology-access-kit/interfaces/semantic-similarity
-
-    Data model:
-
-       https://w3id.org/oak/similarity
-
+    (Function docstring truncated for brevity)
```
Comment (cmungall) [inline src/oaklib/cli.py:2543]: assume this is temporary...?

**Comment by justaddcoffee** [comment]
@cmungall yes - this was just a branch I made while @hrshdhgd and @caufieldjh were testing semsimian/oak stuff

**Comment by justaddcoffee** [comment]
closing since I don't think we need this for testing anymore

**Comment by cmungall** [comment]
If you do need to do a sample for testing with #793 you can specify `.sample` in your query.

**Pull request closed without merging**